src/en/community/meetups/index.html

```diff
@@ -241,7 +241,9 @@ <h3 class="h3">Ceph User + Dev Meetup</h3>
       <a class="a block mb-4" href="https://meet.jit.si/ceph-user-dev-monthly" target="_blank" rel="noreferrer noopener"
         >Join the meeting</a
       >
-      <a class="a" href="https://pad.ceph.com/p/ceph-user-dev-monthly-minutes" target="_blank" rel="noreferrer noopener">Etherpad</a>
+      <a class="a" href="https://pad.ceph.com/p/ceph-user-dev-monthly-minutes" target="_blank" rel="noreferrer noopener">Etherpad</a
+      >
+      <a class="a" href="/{{ locale }}/community/meetups/user-dev-archive" target="_blank" rel="noreferrer noopener">Archive</a>
```

Comment:
It is showing a 404 error in the current format. https://wip-user-dev-files.ceph.io/en/assets/pdfs/user_dev_meeting_2023_09_21_cory_snyder.pdf

Reply:
Will fix

Reply:
It's still 404: https://ceph.io/assets/pdfs/user_dev_meeting_2023_09_21_cory_snyder.pdf

But this is the correct convention, to hardcode https://ceph.io. Once merged, it should point correctly to the file.